2023-11-29-diving-into-jtag-part-2.md

```diff
@@ -0,0 +1,218 @@
+---
+
+title: Diving into JTAG protocol. Part 2 — Debugging.
```

Comment:
Let's match with the first article's title:

```suggestion
title: Diving into JTAG - Debugging (Part 2).
```

Reply:
100% agreed - really appreciate the extra eyes!